packages/core/src/operators/matchPath.operator.ts

```diff
@@ -2,6 +2,8 @@ import * as pathToRegexp from 'path-to-regexp';
 import { Observable } from 'rxjs';
 import { filter, tap } from 'rxjs/operators';
 import { HttpRequest, HttpRoute, RouteParameters } from '../http.interface';
+import { queryParams } from "./queryParams/queryParams.operator";
```

Comment:
Please keep syntax with single quotes

Reply:
I think this should be in `queryParams.factory.ts` file

Reply:
+ I would also indicate in function name that this is a factory `queryParamsFactory`